src/pudl/output/ferc1.py

```diff
@@ -1815,7 +1833,26 @@ def add_sizable_minority_corrections(
         logger.info(
             f"Adding {len(data_corrections)} from {len(bad_utils)} utilities that report differently."
         )
-        return pd.concat([exploded, data_corrections]).reset_index(drop=True)
+        dtype_calced = {
+            col: dtype
+            for (col, dtype) in calculated_df.dtypes.items()
+            if col in data_corrections.columns
+        }
+        corrected = (
+            pd.concat([calculated_df, data_corrections.astype(dtype_calced)])
```

Comment:
Is this to get rid of NA columns for the concatenations?

Reply:
yes! #3125